src/DB_Command.php

```diff
@@ -957,7 +957,7 @@ public function size( $args, $assoc_args ) {
 				}
 					$size_format_display = preg_replace( '/IB$/u', 'iB', strtoupper( $size_format ) );
 
-					$rows[ $index ]['Size'] = ceil( $row['Size'] / $divisor ) . ' ' . $size_format_display;
+					$rows[ $index ]['Size'] = round( $row['Size'] / $divisor ) . ' ' . $size_format_display;
```

Comment:
I think we should add a flag with the number of digits to round to, and default it to `0`. This way, we don't change the current behavior, and people can add their preference to their `wp-cli.yml` config file.

I think the best name for that flag would be `[--decimals=<decimals>]`.

Reply:
@schlessera I added requested chenges but for same reason Behat tests are failing with `Warning: The `wp db size` command has an invalid synopsis part: [--decimals=<decimals>` - see https://github.com/wp-cli/db-command/runs/2992724637?check_suite_focus=true#step:12:27